### ISSUE TITLE
Provide a way to conveniently add `Cookie`s using `{Request,Response}HeadersBuilder`

### ISSUE THREAD

**Issue by ikhoon** [line/armeria#3388]
Currently, users need to explicitly specify a cookie header name for setting a `Cookie` with `{Request,Response}HeadersBuilder`
```java
Cookie = ...;
// For request
RequestHeaders.builder()
              .add(HttpHeaderNames.COOKIE, cookie.toCookieHeader());

// For response
ResponseHeaders.builder()
                .add(HttpHeaderNames.SET_COOKIE, cookie.toSetCookieHeader());
```

It would be convenient if we provide shortcut methods for them.
```java
RequestHeaders.builder().cookie(cookie);
ResponseHeaders.builder().setCookie(cookie);
```

**Comment by heka1024** [comment]
I'll try to add this.

**Comment by ikhoon** [comment]
Thanks in advance! 🙏

**Comment by ikhoon** [comment]
Fixed by #3391
